apps/srt-live-transmit.cpp

```diff
@@ -401,8 +401,12 @@ int main(int argc, char** argv)
     // Set SRT log levels and functional areas
     //
     srt_setloglevel(cfg.loglevel);
-    for (set<srt_logging::LogFA>::iterator i = cfg.logfas.begin(); i != cfg.logfas.end(); ++i)
-        srt_addlogfa(*i);
+    if (!cfg.logfas.empty())
```

Comment:
That makes the usage different to what's in srt-test-live. At least add some explanation at the option help text.

Reply:
`srt-live-transmit` is the main external sample application, and its behavior around FAs is already different from `srt-test-live`.

This PR does not change the behavior much. It just assumes if you provide `-logfa:que-recv`, then you want only this FA to be enabled.
In `srt-test-live` the same will be achieved by disabling `all` FAs first, then enabling the desired FAs, which I find a bit trickier:
`-logfa ~all +que-recv`

Reply:
It's ok that this app works differently. Just explain this in the help text. It's important because it's a user application.